src/test/resources/featurefiles/NavigateValoriWebsite.feature

```diff
@@ -11,3 +11,9 @@ Feature: Test navigation on the valori.nl homepage
     And I navigate to the Mobile Chapter page
     And I contact Valori
     Then the contact us form is available
+    Given fill the contact form with the following data
```

Comment:
Dit zou eigenlijk een nieuw scenario moeten zijn. Het eerste scenario is dat het formulier aanwezig is. Een nieuwe test zou zijn dat je data in de velden kunt invullen en dan valideert of de verwachte data wordt geaccepteerd of niet.  Verder is de Given  nu eigenlijk  een Then.